_data/cat/meshes.yml

```diff
@@ -23,14 +23,23 @@
   multi-tenant: "Experimental"
   desc: "An open platform to connect, monitor, and secure microservices. Governed by Google and IBM."
 
+- name: Kong
+  link: https://konghq.com/solutions/service-mesh/
```

Comment:
Hi @coopr, can we identify a suitable description for Kong Mesh?

`desc:` "Kong - a microservices API gateway."` - - would work for Kong itself. How can we better describe Kong Mesh?

Outside of this description, I'm looking to qualify Kong Mesh as a service mesh. Does Kong Mesh have its own control plane? Does it integration with other meshes, like Istio, as a proxy to displace Envoy?

Reply:
> Kong, the world's most popular OSS API Gateway, now slices, dices, and service meshes!

> Same Kong, new way to use it.

> Clean up your service mess with the Kong service mesh

I'll stop now 😄 

Will update PR now...